client/windows/settings/SettingsMainContainer.cpp

```diff
@@ -0,0 +1,147 @@
+/*
+ * SettingsMainContainer.cpp, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#include <SDL_events.h>
```

Comment:
No need for SDL from what I see.
Also, for includes please use following order:
1) StdInc.h
2) Main include (.h file with the same name as .cpp)
3) VCMI includes
4) Third-party includes
```suggestion
```